src/containers/StructurePage/resourceComponents/TopicDescription.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import Accordion from '../../../components/Accordion';
+import Resource from './Resource';
+import AddArticleModal from './AddArticleModal';
+import { ButtonAppearance } from '../../../components/Accordion/types';
+import { SubjectTopic } from '../../../modules/taxonomy/taxonomyApiInterfaces';
+import { DraftStatus } from '../../../modules/draft/draftApiInterfaces';
+
+interface Props {
+  topicDescription?: string;
+  locale: string;
+  refreshTopics: () => Promise<void>;
+  currentTopic: SubjectTopic;
+  status?: DraftStatus;
+  resourceRef: React.RefObject<any>;
```

Comment:
Skal fungere å putte `HTMLDivElement` inn for any her. https://medium.com/@martin_hotell/react-refs-with-typescript-a32d56c4d315